### DISCUSSION TITLE
Pip index support and python modules for Github Packages

### DISCUSSION THREAD

**Discussion by simonhyll** [community/community#12068] [Actions]
Are there any plans to support Pip and Python for Github Packages? While it's easy to use Github Pages to create an index and you can host packages in github repositories it would be nice to have handling of all organization packages hosted at the same place.

**Comment by orenbenmeir** [comment]
It is an old request that is not being implemented.

A thread from Jan 2020 - https://github.community/t/pypi-compatible-github-package-registry/14615
and another one from 2021 - https://github.com/github-community/community/discussions/8542#discussion-3729602

There is no update from Github on the subject
